src/browser/engine/handler.ts

```diff
@@ -0,0 +1,56 @@
+import { join } from 'path';
+
+const { RpcIpcManager } = require('electron-simple-rpc');
+
+import {
+  Api,
+  Channel,
+  extensionScope,
+  eventScope,
+} from '../../common';
+import { ExtensionEventMessage } from '../../common/types';
+
+export default class Handler<E> {
```

Comment:
Nice pattern 👍 
Add a link to the doc here, or add the doc in the README of engine.

Reply:
Personally I'd prefer documentation in the tsdoc of this class

Reply:
I will add a good tsdoc here